README.md

```diff
@@ -93,7 +93,7 @@ git init
 git add .
```

Comment:
Also, to make it easier, we should encourage them to create their local repo named main too with "git init -b main"

Reply:
You also need a reasonably recent git for `-b`, more recent than the ubuntu default. 🤦